bindings/python/dnet_client

```diff
@@ -130,6 +130,7 @@ def process_backend():
         'enable': lambda x, y, z: process_backend_control(x, y, z, elliptics.Session.enable_backend),
         'disable': lambda x, y, z: process_backend_control(x, y, z, elliptics.Session.disable_backend),
         'defrag': lambda x, y, z: process_backend_control(x, y, z, elliptics.Session.start_defrag),
+        'compact': lambda x, y, z: process_backend_control(x, y, z, elliptics.Session.start_compact),
```

Comment:
ioclient: revert 'start' command for full data defragmentation: https://github.com/abudnik/elliptics/commit/4a4d94cf4fea941e2154367471f8f7dcf6965528